app/src/store/data.ts

```diff
@@ -156,11 +159,33 @@ export default function useDataStore() {
       })
     );
 
+    // test/example
+    const clr = new CLR({
+      calcAlgo: linear,
+      includePayouts: true,
+    } as InitArgs);
+    // get contributions and grantRound details
+    const grantRoundArgs = await fetch({
+      provider: provider.value,
+      grantRound: '0x8B4091997E3EbB87be90cEd3e50d8Bb27e1DC742',
```

Comment:
kind of a nit, just wondering why it's not a constant here.

Reply:
This was just to demo/test that things are working, we will likely want to hook the `prediction` call in rather than `calculate` and we can probably avoid calling `fetch` from the `app` itself too. 

We can remove these calls prior to merging 🙌